src/utils/user.util.js

```
@@ -0,0 +1,7 @@
+export default () => {
+  try {
+    return JSON.parse(localStorage.getItem('user'));
+  } catch (e) {
+    return null;
```

why are you suppressing errors here?

---

It's easier to validate since we expect a valid `user` or `null`.